wasi/wasi.go

```diff
@@ -386,3 +400,9 @@ func environ_sizes_get(*wasm.HostFunctionCallContext, uint32, uint32) (err Errno
 func environ_get(*wasm.HostFunctionCallContext, uint32, uint32) (err Errno) {
 	return
 }
+
+func clock_time_get(ctx *wasm.HostFunctionCallContext, id uint32, precision uint64, timestampPtr uint32) (err Errno) {
```

Comment:
mind adding docs with a correct link for this? here's an example

```
// args_sizes_get is a WASI API that returns the number of the command-line arguments and the total buffer size that
// args_get API will require to store the value of the command-line arguments.
// * argsCountPtr: a pointer to an address of uint32 type. The number of the command-line arguments is written there.
// * argsBufSizePtr: a pointer to an address of uint32 type. The total size of the buffer that the command-line argument data requires is written there.
//
// See: https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#-args_sizes_get---errno-size-size
```

Reply:
Done